docs/user-guide/fhir-resource-tooling.md

```diff
@@ -0,0 +1,42 @@
+
+# FHIR Resource Tooling
+
+There is a tooling project provided for loading necessary resources into a target FHIR server. The project is fhir-resource-tooling and it produces a shaded JAR ```fhir-resource-tooling-VERSION-shaded.jar``` that provides a simple command line interface for loading the resources. The JAR provides two tools, one for loading resources exported from the Measure Authoring Tool (zip files), and one for loading VSAC Value Set spreadsheets as downloaded from https://cts.nlm.nih.gov/fhir.
```

Comment:
I think they refer to the tool as "Cohort and Measure Authoring Tool"  actually on the UI i see it being titled as IBM Watson Health Cohort and Measure Authoring Tool.  MAT = Measure Authoring Tool is the original tool from CDS i think.